include/gatekeeper_gk.h

```diff
@@ -47,12 +49,22 @@ struct gk_config {
 	 * The fields below are for internal use.
 	 * Configuration files should not refer to them.
 	 */
+	unsigned int	   num_tunnels;
 	rte_atomic32_t	   ref_cnt;
 	struct gk_instance *instances;
 	struct net_config  *net;
+
+	/* Specify the Gatekeeper server IP addresses. */
+	struct in_addr     gk_inaddr;
+	struct in6_addr    gk_inaddr6;
+
+	struct ipip_tunnel_info *tunnels;
```

Comment:
Tunnel information is obtained from the LPM table. If a matched prefix says it's protected by Gatekeeper, it will have the information of the tunnel.

Moreover, the MAC addresses must come from the LLS block!